xbmc/interfaces/python/swig.cpp

```diff
@@ -196,9 +196,9 @@ namespace PythonBindings
       PyObject *tracebackModule = PyImport_ImportModule("traceback");
       if (tracebackModule != NULL)
       {
-        PyObject *tbList = PyObject_CallMethod(tracebackModule, "format_exception", "OOO", exc_type, exc_value == NULL ? Py_None : exc_value, exc_traceback == NULL ? Py_None : exc_traceback);
+        PyObject *tbList = PyObject_CallMethod(tracebackModule, (char*)"format_exception", (char*)"OOO", exc_type, exc_value == NULL ? Py_None : exc_value, exc_traceback == NULL ? Py_None : exc_traceback);
```

Comment:
I agree, but the Python C bindings are quite something. FWIW, I read the history of abstract.h and they introduced const in later versions but they cast back to char* in some of the methods anyway (?!?). I would much prefer to bump the binding but that is a much bigger task. I will push ammend this with something I feel better with.